next.config.js

```diff
@@ -9,10 +9,19 @@ const nextConfig = {
     WS_KUSAMA_CORETIME_CHAIN: process.env.WS_KUSAMA_CORETIME_CHAIN || '',
     WS_WESTEND_CORETIME_CHAIN: process.env.WS_WESTEND_CORETIME_CHAIN || '',
 
-    SUBSCAN_CORETIME_ROCOCO_API: process.env.SUBSCAN_CORETIME_ROCOCO_API || '',
-    SUBSCAN_CORETIME_KUSAMA_API: process.env.SUBSCAN_CORETIME_KUSAMA_API || '',
-    SUBSCAN_CORETIME_WESTEND_API:
-      process.env.SUBSCAN_CORETIME_WESTEND_API || '',
+    SUBSCAN_CORETIME_ROCOCO_INDEXER:
+      process.env.SUBSCAN_CORETIME_ROCOCO_INDEXER || '',
```

Comment:
We are no longer using Subscan so it would make sense to rename these variables.